plugins/repository-gcs/licenses/google-NOTICE.txt

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
It looks like we add empty file for libs that don't provide NOTICE.txt (I spotted few others like this in other plugins)